docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+
+## General.TLS
+
+```
+Enabled: false
+# PrivateKey governs the file location of the private key of the TLS certificate.
+PrivateKey: tls/server.key
+# Certificate governs the file location of the server TLS certificate.
+Certificate: tls/server.crt
+RootCAs:
+  - tls/ca.crt
+ClientAuthRequired: false
+ClientRootCAs:
+```
+
+**`Enabled`**: (default value should be overridden) In a production network, you should be using TLS-secured communications. This value should be `true`.
+**`PrivateKey`**: (default value should be overridden). Provide the path to and file name of the private key generated by your TLS CA for this node.
+**`Certificate`**: (default value should be overridden) Provide the path to and filename of the public certificate (also known as the sign certificate) generated by your TLS CA for this node.
+**`RootCAs`**: (should be commented out) This parameter needs to be unset for normal use. It can be used to augment the orderer TLS certificate in the channel configuration orderer `MSPDir/tlscacerts` folder.
+**`ClientAuthRequired`**: (Mutual TLS only) Setting this value to “true” will enable mutual TLS on your network, and must be done for the entire network, not just one node.
+**`ClientRootCAs`**: (Mutual TLS only)  Can be left blank if mutual TLS is disabled. If mutual TLS is enabled, this is the path to the root certificate of the TLS CA.
+
+## General.KeepAlive
+
+The `KeepAlive` values might need to be tuned for compatibility with any networking devices or software (like firewalls or proxies) in between components of your network. Ideally, these settings would be manipulated if needed in a test or pre-prod environment and then set accordingly for your production environment.
+
+```
+# ServerMinInterval is the minimum permitted time between client pings.
+# If clients send pings more frequently, the server will
+# disconnect them.
+ServerMinInterval: 60s
+# ServerInterval is the time between pings to clients.
+ServerInterval: 7200s
+# ServerTimeout is the duration the server waits for a response from
+# a client before closing the connection.
+ServerTimeout: 20s
+```
+
+**`ServerMinInterval`**: (default value should not be overridden, unless determined necessary through testing)
+**`ServerInterval`**: (default value should not be overridden, unless determined necessary through testing)
+**`ServerTimeout`**: (default value should not be overridden, unless determined necessary through testing)
+
+## General.Cluster
+
+```
+# SendBufferSize is the maximum number of messages in the egress buffer.
+# Consensus messages are dropped if the buffer is full, and transaction
+# messages are waiting for space to be freed.
+SendBufferSize: 10
+# ClientCertificate governs the file location of the client TLS certificate
+# used to establish mutual TLS connections with other ordering service nodes.
+ClientCertificate:
+# ClientPrivateKey governs the file location of the private key of the client TLS certificate.
```

Comment:
This comment is being added in PR #2123 :
`# If not set, the server General.TLS.PrivateKey is re-used.`